kite-data/kite-data-hbase/src/main/java/org/kitesdk/data/hbase/HBaseMetadataProvider.java

```diff
@@ -168,6 +171,9 @@ public DatasetDescriptor update(String namespace, String name, DatasetDescriptor
     } else {
       LOG.info("Schema hasn't changed, not migrating: (" + name + ")");
     }
+
+    Map<String, String> properties = getProperties(descriptor);
```

Comment:
Yea that makes sense. But was not sure if that was intentionally avoided before with the listProperties() and getProperty() as the alternative. I will add a properties(Map<String, String> to the Descriptor builder and a getProperties() to the Descriptor